pkgs/by-name/j/j/package.nix

```
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "jsoftware";
     repo = "jsource";
-    rev = "${version}";
+    tag = "${version}";
```

If this line is getting tweaked, might as well fix this too:

```suggestion
    tag = version;
```

---

Nice, I missed that one.